### DISCUSSION TITLE
LSP config

### DISCUSSION THREAD

**Discussion by jin-ahn** [LunarVim/LunarVim#2126] [Q&A] [answered]
Trying to use this plugin https://github.com/stevearc/aerial.nvim#installation

As part of the configuration it says i need add the following to my LSP config
```
-- Set up your LSP clients here, using the aerial on_attach method
require'lspconfig'.vimls.setup{
  on_attach = aerial.on_attach,
}
```

How do i do this in my config.lua for lunar vim?

**Comment by kylo252** [comment, marked as answer]
Check this section here: https://www.lunarvim.org/languages/#overriding-the-default-setup-options
